docs/assets/css/sidebar_doc.css

```diff
@@ -1,140 +1,136 @@
 .row .height-100 {
-    height: auto;
-    padding: 0;
+  height: auto;
+  padding: 0;
 }
 .doc-heading {
-    font-size: 16px;
-    font-weight: 600;
-    text-transform: uppercase;
-    white-space: nowrap;
+  font-size: 16px;
+  font-weight: 600;
+  text-transform: uppercase;
+  white-space: nowrap;
 }
 .doc-link {
-    font-size: 16px;
-    text-transform: capitalize;
-    color: #000;
+  font-size: 16px;
+  text-transform: capitalize;
+  color: #000;
 }
 .accordion-collapse li h5 {
-    margin-bottom: 0;
+  margin-bottom: 0;
 }
 .accordion-item {
-    border-bottom: none !important;
-    padding: 0 !important;
+  border-bottom: none !important;
+  padding: 0 !important;
 }
 .navbar {
-    height: 100%;
-    margin-top: -30px ;
-    position: relative;
+  height: 100%;
+  margin-top: -30px;
+  position: fixed;
 }
 
 .sidebar {
-    position: absolute;
-    top: 30px;
+  position: absolute;
```

Comment:
Het @ritesh-karankal you have to change position of this class, also increase the top also

Reply:
```suggestion
.sidebar{
  position: fixed;
  top: 145px;

```
Is it correct?

Reply:
Yeah!! looks good @ritesh-karankal